src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2169,16 +2185,19 @@ internal void GenerateAnonymousFunctionConversionError(BindingDiagnosticBag diag
 
             if (reason == LambdaConversionResult.MismatchedParameterType)
             {
+                // This is checked in the code that returns LambdaConversionResult.MismatchedParameterType
+                Debug.Assert(anonymousFunction.HasExplicitlyTypedParameterList);
+
                 // Cannot convert {0} to type '{1}' because the parameter types do not match the delegate parameter types
+
                 conversionError(diagnostics, ErrorCode.ERR_CantConvAnonMethParams, id, targetType);
```

Comment:
Please remove the blank line since the comment applies to the `conversionError()` call.

Reply:
done.